### PR TITLE
bump inspec version to 4.7.18

### PR DESCRIPTION
**Pull request by vjeffrey**
### :nut_and_bolt: Description
just bumping the version of inspec used.
ran an aws api scan locally with this version to ensure everything works as expected (https://github.com/chef/automate/issues/75)

also: i'm not certain why expeditor didn't auto-open a pr for us, but it looks like that hasn't happened since we moved repos, so maybe need to update a repo reference over in inspec's ci pipeline.

### CODE REVIEW

**Comment by vjeffrey** [comment]
hmm. the version bump is definitely causing some failures in our tests. i'm going to mark this as wip while i take a look at these.

**Comment by stevendanna** [comment]
If you modify this: https://github.com/chef/automate/blob/master/.expeditor/config.yml#L162-L163 follow the 4.x series, expeditor will keep us updated

**Comment by phiggins** [comment]
Somewhat unrelated to this PR, but I looked for other occurrences of `3.9.0` and found this sorta inaccurate comment. https://github.com/chef/automate/blob/master/api/config/compliance/config_request.go#L34-L35

The value does come from `default.toml`, but that file gets its value from the `plan.sh`. This comment makes it sound like there is another place to update in PRs like this one.

File: .studio/chef-server-collection [hunk 1]
```diff
@@ -93,8 +93,8 @@ EOH
     "reporter": "chef-server-automate",
     "fetcher": "chef-server",
     "profiles": [{
-      "name": "apache-baseline",
-      "compliance": "test/apache-baseline"
+      "name": "ssl-baseline",
```
Comment (lancewf) [inline .studio/chef-server-collection]: I am just curious why the change from apache-baseline to ssl-baseline.

Comment (vjeffrey) [reply .studio/chef-server-collection]: that profile is using some deprecated resources that are causing the inspec check and exec calls to fail
